### ISSUE TITLE
Translations for default language are not working, when default language is not English

### ISSUE THREAD

**Issue by romanustin** [youngcut/kirby-form-block-suite#6]
My default and only language is not English. When I add other languages, translations are working fine, but when I am adding translations for default and only language, here is what I see:

<img width="829" alt="image" src="https://user-images.githubusercontent.com/53936318/210388648-29d9b99f-803e-402e-a398-363828983d20.png">

**Comment by youngcut** [comment]
Thanks for report. What's your default language?

**Comment by youngcut** [comment]
Or the language of your panel.

**Comment by romanustin** [comment]
Yes, to be more precise, I mean language of panel. In my case it's Lithuanian, code "lt"

**Comment by youngcut** [comment]
Okay. That language is not support. You can copy the translation-file from `site/plugins/kirby-form-block-suite/lib/languages/de.php` to `site/plugins/kirby-form-block-suite/lib/languages/lt.php` and load it on the end of `site/plugins/kirby-form-block-suite/index.php` like this:
``` php

'translations' => [
   'en' => require __DIR__ . '/lib/languages/en.php',
   'de' => require __DIR__ . '/lib/languages/de.php'
   'lt' => require __DIR__ . '/lib/languages/lt.php',
]
```

**Comment by youngcut** [comment]
I thing, that's a bug of kirby. I made [a forum entry](https://forum.getkirby.com/t/plugin-language-missing-lt/27441) for you.

**Comment by romanustin** [comment]
Also "Submit" button in frontend doesn't have any text.

**Comment by youngcut** [comment]
Thanks for translating. If you don't mind to translate[ the defaults](https://github.com/youngcut/kirby-form-block-suite/tree/main/config) too. I'm look forward to your pr.
